source/install/cloud-changelog.md

```diff
@@ -3,12 +3,38 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2021-12-08](#release-2021-12-08)
 - [Release 2021-12-01](#release-2021-12-01)
 - [Release 2021-11-23](#release-2021-11-23)
 - [Release 2021-11-11](#release-2021-11-11)
 - [Release 2021-11-10](#release-2021-11-10)
 - [Release 2021-10-27](#release-2021-10-27)
-- [Release 2021-10-13](#release-2021-10-13)
+
+## Release 2021-12-08
+
+### Improvements
+
+#### User Interface (UI)
+ - Added thread replies to search results when collapsed reply threads feature is enabled.
+ - Updated "Terms of Service" terminology to "Terms of Use" product-wide.
+ - Government related references are now shown for Government type licenses.
+
+#### Performance
+ - Added a general performance fix for loading the web application and typing.
+ - Improved performance while typing by moving some autocomplete layout calculation.
```

Comment:
```suggestion
 - Improved performance while typing by moving some autocomplete layout calculations.
```

Reply:
I'm not 100% sure that pluralizing the word "calculations" is technical accurate (where multiple calculations were impacted by a change); I pluralized it to make the sentence flow a bit better.